framework/components/ecl-messages/_messages.scss

```diff
@@ -1,120 +1,63 @@
-
-.messages {
-  position: relative;
-  min-height: 5.375em;
-  margin: 0 0 1em;
-  padding: 1.315em 0 1.315em 5em;
-  overflow: hidden;
-  background: transparent url('../images/svg/messages-info.svg') no-repeat 1.375em 1.375em;
-  font-family: $ecl-font-family-sans-serif;
-  font-size: $messages-text-size;
-  color: $messages-text-color;
+.ecl-message {
+  background: transparent url('../images/svg/messages-info.svg') no-repeat 1.2rem 1.2rem;
   border: 2px solid map-get($ecl-colors, 'blue-75');
+  color: $ecl-color-shade;
+  font-size: map-get($ecl-font-size, 's');
+  margin-bottom: map-get($ecl-spacing, 's');
+  min-height: map-get($ecl-spacing, 'l');
+  padding: map-get($ecl-spacing, 'xs') map-get($ecl-spacing, 's') map-get($ecl-spacing, 'xs') map-get($ecl-spacing, 'xxl');
+  position: relative;
 
-  &.status {
+  &--success {
     background-image: url('../images/svg/messages-success.svg');
     border-color: $ecl-color-success;
   }
 
-  &.warning {
+  &--warning {
     background-image: url('../images/svg/messages-warning.svg');
     border-color: map-get($ecl-colors, 'yellow-110');
   }
 
-  &.error {
+  &--error {
     background-image: url('../images/svg/messages-error.svg');
     border-color: $ecl-color-error;
   }
 
-  &.live {
+  &--live {
     background-image: url('../images/svg/live_streaming.svg');
     border-color: map-get($ecl-colors, 'yellow-110');
   }
-
-  h3 {
-    padding-top: 0;
-    font-size: inherit;
-    font-weight: bold;
-  }
-
-  p {
-    max-width: 95%;
-    margin-bottom: 0;
-    font-size: $messages-text-size;
-  }
-
-  > :last-child {
-    padding-bottom: 0;
-  }
-
-  .close {
-    display: none;
-  }
 }
 
-// Bootstrap overrides.
-.alert {
-  margin: 1.667em 0;
-  border-radius: 0;
-
-  .close {
-    display: block;
-    position: absolute;
-    top: 0.465em;
-    right: 0.465em;
-    padding: 0 0 0 5px;
-    background: transparent url('../images/svg/close.svg') no-repeat right top;
-    font-size: 1.625em;
-    color: transparent;
-    text-shadow: none;
-    text-decoration: none;
-    opacity: 1;
-
-    &:hover {
-      background-image: url('../images/svg/close_hover.svg');
-    }
-  }
+.ecl-message__title {
+  font-weight: bold;
+  margin-bottom: map-get($ecl-spacing, 'xs');
 }
 
-.no-svg {
-  .messages {
-    background-image: url('../images/png/messages-info.png');
-
-    &.status {
-      background-image: url('../images/png/messages-info.png');
-    }
-
-    &.warning {
-      background-image: url('../images/png/messages-warning.png');
-    }
-
-    &.error {
-      background-image: url('../images/png/messages-error.png');
-    }
-
-    &.live {
-      background-image: url('../images/png/live_streaming.png');
-    }
-  }
-
-  .alert {
-    .close {
-      background-image: url('../images/png/close.png');
+.ecl-message__body {
+  margin: 0;
+}
 
-      &:hover {
-        background-image: url('../images/png/close_hover.png');
-      }
-    }
+.ecl-message__dismiss {
```

Comment:
Here I'd explicitly declare `width` and `height` and a background-size: contain`. No need for `line-height` or `font-size` then